client/rest-high-level/src/test/java/org/elasticsearch/client/SnapshotIT.java

```diff
@@ -285,6 +286,49 @@ public void testRestoreSnapshot() throws IOException {
         assertThat(restoreInfo.failedShards(), equalTo(0));
     }
 
+    public void testSnapshotHidden() throws IOException {
```

Comment:
This test failed because the `.security` index could not be restored (either because it was never deleted as part of the test or auto re-created).

Reply:
Thanks, I believe I've fixed this test - there were a few issues with it.